code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1110,6 +1110,14 @@
 	..()
 	. = TRUE
 
+/datum/reagent/medicine/antihol/expose_mob(mob/living/carbon/exposed_carbon, methods=TOUCH, reac_volume)
+	. = ..()
+	if(!(methods & (TOUCH|VAPOR|PATCH)))
+		return
+
+	for(var/datum/surgery/surgery as anything in exposed_carbon.surgeries)
+		surgery.speed_modifier -= 0.1
```

Comment:
```suggestion
		surgery.speed_modifier = max(surgery.speed_modifier  - 0.1, -0.9)
```
a speed modifier of -1 will lead to division by zero, and any lower than that, to a negative time.